jvm/pom.xml

```diff
@@ -505,6 +505,7 @@
             <plugin>
                 <groupId>org.scalatest</groupId>
                 <artifactId>scalatest-maven-plugin</artifactId>
+                <version>${scalatest-maven-plugin.version}</version>
```

Comment:
Maybe you can try `PluginManagement` to avoid specify version in each module https://github.com/oap-project/gluten/blob/main/pom.xml#L499